tests/suites/test_suite_psa_crypto_driver_wrappers.function

```diff
@@ -1983,6 +1983,8 @@ void asymmetric_decrypt( int alg_arg,
         mbedtls_test_driver_rsa_hooks.forced_output_length = fake_output_decrypt->len;
         output_size = fake_output_decrypt->len;
         ASSERT_ALLOC( output, output_size );
+
+        expected_output_data = fake_output_decrypt;
```

Comment:
What I meant is even going a step further - just assign the values in the test data, and remove this line. This way it won't be surprising for anyone that they are setting `expected_output_data` in test data, and it's not used, as will be the case with this code.